README.md

```diff
@@ -95,6 +119,14 @@ What are the differences between calling and referencing a function? Please prov
 
 ```text
 
+To be honest I'm a little confused by what the difference between the two is but this is what I think it is:
+
+var test=function(){    // this is referencing a function 
+
+}
+
+test();  // this is calling a function
```

Comment:
:+1:  calling the function is correct.  Can you give me an example of we commonly reference a function, rather than calling/executing it?